src/components/MovieCard/moviecard.js

```diff
@@ -1,18 +1,54 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
+import ContextualMenu from '../ContextualMenu/contextualmenu';
+import more from '../../img/more.svg';
 
 import './moviecard.css';
 
-const MovieCard = ({ title, description, year }) => (
-  <div className="movie-card">
-    <img className="movie-image" src="https://i.pinimg.com/originals/f3/a2/0d/f3a20d7df90d3b4a4167a419a0566ff3.jpg" alt="Not available" />
-    <div className="image-footer">
-      <h3>{title}</h3>
-      <div>{year}</div>
+const MovieCard = ({ title, description, year }) => {
+  const [menuState, setMenuState] = useState({
+    xPos: '0px',
+    yPos: '0px',
+    showMenu: false,
+  });
+  
+  const handleClick = (e) => {
+    if (menuState.showMenu) setMenuState({ showMenu: false });
+  };
+  
+  useEffect(() => {
+    window.addEventListener('click', handleClick);
+    
+    return () => {
+      window.removeEventListener('click', handleClick);
+    };
+  });
+  
+  const handleContextMenu = (e) => {
+    e.preventDefault();
+    
+    setMenuState({
+      xPos: `${e.pageX}px`,
+      yPos: `${e.pageY}px`,
```

Comment:
I think that can be implemented without setting the coordinates manually. With a proper layout your context menu should be placed in a right place without having style. Please take a look at that

Reply:
should I use another component for that? not sure how to do that

Reply:
fixed